tmpl/volk_cpu.tmpl.c

```diff
@@ -24,189 +24,33 @@
 #include <stdlib.h>
 #include <string.h>
 
```

Comment:
Need
```
#if defined(__cplusplus)
using namespace cpu_features;
#endif
```
here to get this compiling with MSVC, where we are compiling the source as C++ to accommodate missing `complex.h` with C and MSVC. When the `cpu_features` headers are included from C++, they put everything in the `cpu_features` namespace.